locales/en/fluid-queue.json

```diff
@@ -61,9 +61,10 @@
   "unknownEntry": "unknown entry",
   "queueEmptyNext": "The queue is empty.",
   "timerExpired": "{{channel}} the timer has expired for this level!",
-  "nowPlayingBasic": "Now playing {{level}} submitted by {{level.submitter}}.",
-  "nowPlayingWeightedRandom": "Now playing {{level}} submitted by {{- level.submitter}} with a {{level.selectionChance}}%{{percentSuffix}} chance of selection.",
-  "nowPlayingWeightedNext": "Now playing {{level}} submitted by {{- level.submitter}} with the highest wait time of {{level.selectionChance}}%{{percentSuffix}}.",
+  "userOffline": " This user does not appear to be online.",
+  "nowPlayingBasic": "Now playing {{level}} submitted by {{level.submitter}}.{{offline}}",
+  "nowPlayingWeightedRandom": "Now playing {{level}} submitted by {{- level.submitter}} with a {{level.selectionChance}}%{{percentSuffix}} chance of selection. {{offline}}",
+  "nowPlayingWeightedNext": "Now playing {{level}} submitted by {{- level.submitter}} with the highest wait time of {{level.selectionChance}}%{{percentSuffix}}. {{offline}}",
```

Comment:
`nowPlayingBasic` does not have a space after the `.`, but `nowPlayingWeightedRandom` and `nowPlayingWeightedNext` have a space between `.` and `{{offline}}`.
This does not matter for twitch chat, since two spaces are converted to one (and a trailing space might not matter).
However I think for testing these bot responses it might be good to not introduce 2 spaces.